.gitignore

```diff
@@ -68,4 +68,7 @@ target/
 data/
 
 # Editor / IDE
-.vscode
\ No newline at end of file
+.vscode
+
+venv
+addresses.txt
```

Comment:
I don't think we should add addresses.txt to gitignore - why should it be in the project in the first place?